gravityflow/gravityflowchecklists
# Fix incomplete submissions table name

This PR fixes an issue that with Gravity Forms 2.4.x, when set a form enabling "Save and Continue" in the Checklists, there is a PHP notice thrown and the incomplete submission cannot be linked correctly.

## Testing instructions
1. Create a random form and enable "Save and Continue."
2. Create a new checklist and use that form.
3. Fill up parts of the form and click on "Save and Continue Later."
4. On the `master` branch, a PHP notice will be thrown and the incomplete submission cannot be retrieved. Switch to this branch can get it to work.
